[2023-10-03T10:10:25.892+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-24T00:00:00+00:00 [queued]>
[2023-10-03T10:10:25.972+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-24T00:00:00+00:00 [queued]>
[2023-10-03T10:10:25.973+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-10-03T10:10:26.199+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): check_data_in_a_day> on 2023-09-24 00:00:00+00:00
[2023-10-03T10:10:26.293+0000] {standard_task_runner.py:57} INFO - Started process 3072 to run task
[2023-10-03T10:10:26.369+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'launches_getter', 'check_data_in_a_day', 'scheduled__2023-09-24T00:00:00+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/launches.py', '--cfg-path', '/tmp/tmp08bivys7']
[2023-10-03T10:10:26.377+0000] {standard_task_runner.py:85} INFO - Job 178: Subtask check_data_in_a_day
[2023-10-03T10:10:26.666+0000] {task_command.py:410} INFO - Running <TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-24T00:00:00+00:00 [running]> on host f21dc12153d8
[2023-10-03T10:10:27.024+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='launches_getter' AIRFLOW_CTX_TASK_ID='check_data_in_a_day' AIRFLOW_CTX_EXECUTION_DATE='2023-09-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-24T00:00:00+00:00'
[2023-10-03T10:10:27.110+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-10-03T10:10:27.139+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=launches_getter, task_id=check_data_in_a_day, execution_date=20230924T000000, start_date=20231003T101025, end_date=20231003T101027
[2023-10-03T10:10:27.328+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-10-03T10:10:27.462+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-03T12:31:48.424+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-24T00:00:00+00:00 [queued]>
[2023-10-03T12:31:48.537+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-24T00:00:00+00:00 [queued]>
[2023-10-03T12:31:48.539+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-10-03T12:31:48.684+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): check_data_in_a_day> on 2023-09-24 00:00:00+00:00
[2023-10-03T12:31:48.716+0000] {standard_task_runner.py:57} INFO - Started process 194 to run task
[2023-10-03T12:31:48.735+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'launches_getter', 'check_data_in_a_day', 'scheduled__2023-09-24T00:00:00+00:00', '--job-id', '275', '--raw', '--subdir', 'DAGS_FOLDER/launches.py', '--cfg-path', '/tmp/tmp8dfrlvmr']
[2023-10-03T12:31:48.752+0000] {standard_task_runner.py:85} INFO - Job 275: Subtask check_data_in_a_day
[2023-10-03T12:31:49.062+0000] {task_command.py:410} INFO - Running <TaskInstance: launches_getter.check_data_in_a_day scheduled__2023-09-24T00:00:00+00:00 [running]> on host f21dc12153d8
[2023-10-03T12:31:49.810+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='launches_getter' AIRFLOW_CTX_TASK_ID='check_data_in_a_day' AIRFLOW_CTX_EXECUTION_DATE='2023-09-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-24T00:00:00+00:00'
[2023-10-03T12:31:49.910+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-10-03T12:31:49.957+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=launches_getter, task_id=check_data_in_a_day, execution_date=20230924T000000, start_date=20231003T123148, end_date=20231003T123149
[2023-10-03T12:31:50.143+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-10-03T12:31:50.250+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
